psignifit/psignifit.py

```diff
@@ -81,22 +81,25 @@ def psignifit(data: np.ndarray, conf: Optional[Configuration] = None,
     if conf.fixed_parameters is not None:
         for param, value in conf.fixed_parameters.items():
             bounds[param] = (value, value)
-
     priors = setup_priors(custom_priors=conf.priors, bounds=bounds,
                           stimulus_range=stimulus_range, width_min=width_min, width_alpha=conf.width_alpha,
                           beta_prior=conf.beta_prior, threshold_perc_correct=conf.thresh_PC)
     fit_dict, posteriors, grid = _fit_parameters(data, bounds, priors, sigmoid, conf.steps_moving_bounds,
                                                  conf.max_bound_value, conf.grid_steps)
 
     grid_values = [grid_value for _, grid_value in sorted(grid.items())]
-    intervals = confidence_intervals(posteriors, grid_values, conf.confP, conf.CI_method)
+    intervals = confidence_intervals(posteriors, grid_values, conf.confidence_percentiles, conf.CI_method)
     intervals_dict = {param: interval_per_p.tolist()
                       for param, interval_per_p in zip(sorted(grid.keys()), intervals)}
     marginals = marginalize_posterior(grid, posteriors)
 
     if conf.verbose:
         _warn_marginal_sanity_checks(marginals)
 
+    if 'gamma' not in fit_dict:  # equal asymptotes
+        fit_dict['gamma'] = fit_dict['lambda']
+        intervals_dict['gamma'] = intervals_dict['lambda']
```

Comment:
this is taken care of, at least partially in the following lines (106-110), so line 101
`intervals_dict['gamma'] = intervals_dict['lambda']` should be added after line 110 below

Reply:
Actually, given its age this PR is too full and does too many things. I currently see it more as a playground to discover if/why there are differenes between the MATLAB and Python version. Afterwards IMHO the essence of this PR should be reimplemented in a separate branch.